### ISSUE TITLE
Why using cosine to calculate loss instead of the squared error?

### ISSUE THREAD

**Issue by zhaoqf123** [lsj2408/Transformer-M#17]
Thank you for your excellent work!

May I know why you use cosine similarity to calculate loss instead of the squared error as in the paper `PRE-TRAINING VIA DENOISING FOR MOLECULAR PROPERTY PREDICTION`?
